<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>533</width>
    <height>645</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>DayZ Unix Launcher</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="layout_vertical_main">
      <item>
       <widget class="QTabWidget" name="widget_tabs">
        <property name="sizePolicy">
         <sizepolicy hsizetype="MinimumExpanding" vsizetype="MinimumExpanding">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>500</width>
          <height>500</height>
         </size>
        </property>
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="tab_mods">
         <attribute name="title">
          <string>Mods</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_2">
          <item>
           <widget class="QLabel" name="label_workshop_status">
            <property name="font">
             <font>
              <pointsize>9</pointsize>
             </font>
            </property>
            <property name="text">
             <string>Steam Workshop - idle</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignCenter</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="ModTableWidget" name="table_mods"/>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_horizontal_buttons_add_remove">
            <item>
             <widget class="QLabel" name="label_custom_mods_buttons">
              <property name="text">
               <string>Custom mods:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_add_custom_mod">
              <property name="text">
               <string>Add</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/open-iconic/plus.svg</normaloff>:/icons/open-iconic/plus.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_remove_custom_mod">
              <property name="text">
               <string>Remove</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/open-iconic/minus.svg</normaloff>:/icons/open-iconic/minus.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_horizontal_mod_preset">
            <item>
             <widget class="QLabel" name="label_mod_preset">
              <property name="text">
               <string>Mod preset:</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_mod_preset_open">
              <property name="text">
               <string>Open</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/open-iconic/envelope-open.svg</normaloff>:/icons/open-iconic/envelope-open.svg</iconset>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="button_mod_preset_save">
              <property name="text">
               <string>Save</string>
              </property>
              <property name="icon">
               <iconset resource="icons.qrc">
                <normaloff>:/icons/open-iconic/envelope-closed.svg</normaloff>:/icons/open-iconic/envelope-closed.svg</iconset>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_horizontal_selected_mods">
            <item>
             <widget class="QLabel" name="label_selected_mods">
              <property name="frameShape">
               <enum>QFrame::Box</enum>
              </property>
              <property name="lineWidth">
               <number>0</number>
              </property>
              <property name="text">
               <string>Selected 0 mods (0 from workshop, 0 custom)</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <layout class="QHBoxLayout" name="layout_horizontal_button_start_lbl_status">
            <item>
             <widget class="QPushButton" name="button_start">
              <property name="text">
               <string>Start</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="label_arma_status">
              <property name="text">
               <string>Status: DayZ is not running</string>
              </property>
              <property name="alignment">
               <set>Qt::AlignCenter</set>
              </property>
             </widget>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_parameters">
         <attribute name="title">
          <string>Parameters</string>
         </attribute>
         <layout class="QGridLayout" name="gridLayout_2">
          <item row="0" column="0">
           <widget class="QTabWidget" name="widget_tabs_parameters">
            <property name="tabShape">
             <enum>QTabWidget::Rounded</enum>
            </property>
            <property name="currentIndex">
             <number>0</number>
            </property>
            <property name="tabBarAutoHide">
             <bool>true</bool>
            </property>
            <widget class="QWidget" name="tab_parameters_basic">
             <attribute name="title">
              <string>Basic</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_4">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="layout_vertical_parameters_basic">
                <item>
                 <widget class="QCheckBox" name="checkbox_skip_intro">
                  <property name="text">
                   <string>Skip intro</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_skip_logos_at_startup">
                  <property name="text">
                   <string>Skip logos at startup</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_force_window_mode">
                  <property name="text">
                   <string>Force window mode</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_basic_profile">
                  <item>
                   <widget class="QCheckBox" name="checkbox_profile">
                    <property name="text">
                     <string>Profile</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_profile">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_dlc_contact">
                  <property name="text">
                   <string>Enable Contact DLC</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_dlc_global_mobilization">
                  <property name="text">
                   <string>Enable Global Mobilization - Cold War Germany DLC</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_dlc_sog_prairie_fire">
                  <property name="text">
                   <string>Enable S.O.G. Prairie Fire DLC</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_dlc_csla">
                  <property name="text">
                   <string>Enable CSLA Iron Curtain DLC</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_dlc_western_sahara">
                  <property name="text">
                   <string>Enable Western Sahara DLC</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_disable_esync">
                  <property name="text">
                   <string>Proton: Disable esync</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QVBoxLayout" name="layout_vertical_parameters_basic_environment_variables">
                  <item>
                   <widget class="QCheckBox" name="checkbox_environment_variables">
                    <property name="text">
                     <string>Environment variables</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_environment_variables">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
               </layout>
              </item>
              <item row="1" column="0">
               <spacer name="verticalSpacer">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_parameters_advanced">
             <attribute name="title">
              <string>Advanced</string>
             </attribute>
             <layout class="QVBoxLayout" name="verticalLayout_8">
              <item>
               <layout class="QVBoxLayout" name="layout_vertical_parameters_advanced">
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_advanced_parameter_file">
                  <item>
                   <widget class="QCheckBox" name="checkbox_parameter_file">
                    <property name="text">
                     <string>Parameter file</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_parameter_file">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QPushButton" name="button_parameter_file_open">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="text">
                     <string>Open</string>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_check_signatures">
                  <property name="text">
                   <string>Check signatures</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_advanced_cpu_count">
                  <item>
                   <widget class="QCheckBox" name="checkbox_cpu_count">
                    <property name="text">
                     <string>CPU count</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QSpinBox" name="spinbox_cpu_count">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                    <property name="minimum">
                     <number>1</number>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_advanced_extra_threads">
                  <item>
                   <widget class="QCheckBox" name="checkbox_extra_threads">
                    <property name="text">
                     <string>Extra threads</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <layout class="QVBoxLayout" name="layout_vertical_parameters_advanced_extra_threads">
                    <item>
                     <widget class="QCheckBox" name="checkbox_extra_threads_file_operations">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>File operations</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_extra_threads_texture_loading">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Texture loading</string>
                      </property>
                     </widget>
                    </item>
                    <item>
                     <widget class="QCheckBox" name="checkbox_extra_threads_geometry_loading">
                      <property name="enabled">
                       <bool>false</bool>
                      </property>
                      <property name="text">
                       <string>Geometry loading</string>
                      </property>
                     </widget>
                    </item>
                   </layout>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_enable_hyper_threading">
                  <property name="text">
                   <string>Enable Hyper-Threading</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_enable_file_patching">
                  <property name="text">
                   <string>Enable file patching</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_no_logs">
                  <property name="text">
                   <string>No logs</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_advanced_world">
                  <item>
                   <widget class="QCheckBox" name="checkbox_world">
                    <property name="text">
                     <string>World</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_world">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_no_pause">
                  <property name="text">
                   <string>No pause</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_hugepages">
                  <property name="text">
                   <string>Huge pages</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_advanced_custom_parameters">
                  <item>
                   <widget class="QCheckBox" name="checkbox_custom_parameters">
                    <property name="text">
                     <string>Custom parameters</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_custom_parameters">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <spacer name="verticalSpacer_2">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
            <widget class="QWidget" name="tab_parameters_client">
             <attribute name="title">
              <string>Client</string>
             </attribute>
             <layout class="QGridLayout" name="gridLayout_3">
              <item row="0" column="0">
               <layout class="QVBoxLayout" name="layout_vertical_parameters_client">
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_client_server_address">
                  <item>
                   <widget class="QCheckBox" name="checkbox_server_address">
                    <property name="text">
                     <string>Server address</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_server_address">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_client_server_port">
                  <item>
                   <widget class="QCheckBox" name="checkbox_server_port">
                    <property name="text">
                     <string>Server port</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_server_port">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <layout class="QHBoxLayout" name="layout_horizontal_parameters_client_server_password">
                  <item>
                   <widget class="QCheckBox" name="checkbox_server_password">
                    <property name="text">
                     <string>Server password</string>
                    </property>
                   </widget>
                  </item>
                  <item>
                   <widget class="QLineEdit" name="textbox_server_password">
                    <property name="enabled">
                     <bool>false</bool>
                    </property>
                   </widget>
                  </item>
                 </layout>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkbox_host_session">
                  <property name="text">
                   <string>Host session</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <spacer name="verticalSpacer_3">
                  <property name="orientation">
                   <enum>Qt::Vertical</enum>
                  </property>
                  <property name="sizeHint" stdset="0">
                   <size>
                    <width>20</width>
                    <height>40</height>
                   </size>
                  </property>
                 </spacer>
                </item>
               </layout>
              </item>
             </layout>
            </widget>
           </widget>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="tab_other">
         <attribute name="title">
          <string>Other</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="button_export_workshop_to_txt">
            <property name="text">
             <string>Export Workshop mods to .txt file</string>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QHBoxLayout" name="horizontalLayout_2">
            <item>
             <widget class="QLabel" name="label_theme">
              <property name="text">
               <string>Theme</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QComboBox" name="combobox_theme">
              <property name="sizePolicy">
               <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
             </widget>
            </item>
           </layout>
          </item>
          <item>
           <widget class="QPushButton" name="button_quit">
            <property name="text">
             <string>Quit</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer_4">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>470</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <action name="action_mods_disable_all">
   <property name="text">
    <string>Disable all mods</string>
   </property>
   <property name="toolTip">
    <string>Disable all mods</string>
   </property>
  </action>
  <action name="action_mods_save_json">
   <property name="text">
    <string>Save JSON preset</string>
   </property>
  </action>
  <action name="action_mods_save_html">
   <property name="text">
    <string>Save HTML preset</string>
   </property>
   <property name="toolTip">
    <string>Save HTML preset</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>ModTableWidget</class>
   <extends>QTableWidget</extends>
   <header location="global">modtablewidget.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="icons.qrc"/>
 </resources>
 <connections/>
</ui>
